testutils/logrus_hook.go

```diff
@@ -0,0 +1,59 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2019 Load Impact
```

Comment:
How do we handle the copyright header for files copied from k6, or well any other project? We have to keep it intact, right? So do we add the xk6-browser one below?

@robingustafsson

In this case we could import `lib.testutils.SimpleLogrusHook` directly from k6, but since this is a test helper and might change I'd rather we copy it.

Reply:
@imiric If we copy them from projects under the /grafana gh org and the projects have CLAs we are free to re-license the code but I'd suggest we follow these guidelines:

1. If the repo we're copying code from is a repo under /grafana gh org and it has the same license as xk6-browser we can switch to the xk6-browser copyright header.
2. If the repo we're copying code from is a repo under /grafana gh org but has a different license from xk6-browser (will always be more liberal than AGPLv3) we retain the license from the source repo and add both the license header from the source as well as the xk6-browser license header to cover any modifications.